docs/source/chapt_pyomo/tutorial/pyomo_test.rst

```diff
@@ -23,51 +23,44 @@ Instructions
+   Enter the following PYOMO code for the optimization model:
+   code:: python
+
+  * ``1. from pyomo.environ import (Var,Constraint,ConcreteModel,value,Param)``
+  * ``2. from pyomo.opt import SolverFactory``
+  * ``3. import pyutilib.subprocess.GlobalData ``
+  * ``4. pyutilib.subprocess.GlobalData.DEFINE_SIGNAL_HANDLERS_DEFAULT = False ``
+  * ``5. m=ConcreteModel() ``
+  * ``6. m.x1=Var(within=PositiveReals) ``
+  * ``7. m.x2=Var(within=PositiveReals) ``
+  * ``8. m.y=Var() ``
+  * ``9. m.c1=Constraint(expr=x["a"]*m.x1+x["b"]*m.x2>=x["c"]) ``
+  * ``10.m.c2=Constraint(expr=m.x1+m.x2==m.y) ``
+  * ``11.m.o=Objective(expr=m.y)``
+  * ``12.opt = SolverFactory("ipopt") ``
+  * ``13.opt.solve(m) ``
+  * ``14.f["x1"]=m.x1.value ``
+  * ``15.f["x2"]=m.x2.value ``
+  * ``16.f["y"]=m.y.value ``
 
 In the above code, lines (1), (2) are used to import the PYOMO package and SolverFactory function to develop the model and solve it by accessing an appropriate solver.
 
-The solvers can be accessed in 2 ways:
-(i) PYOMO Solver Plugins: directly accessing the solvers available within Pyomo, like glpk, cplex, baron, etc.
-
-Eg: Opt = SolverFactory(“glpk”).
-
-(ii) PYOMO GAMS Interface: In case some of the licensed solvers like BARON or CPLEX are not directly available to users, they can be accessed in PYOMO through GAMS.
+A PYOMO Concrete Model is declared, defining the variables, declaring the constraints using the parameters defined within “Input Variables” of the Node, and defining the objective function. (Lines (5) to (11)).
 
-More information on the syntax can be found in the following link: https://pyomo.readthedocs.io/en/latest/library_reference/solvers/gams.html.
+Lines (12) & (13) are used to access the solver, and solve the model via ipopt solver.
 
-In order to allow a subprocess to be created for solving the model on another thread apart from the main one, lines (3) & (4) are used to set the default signal handler within pyutilib package of PYOMO to “false” (ie: removing it).
+Note: ipopt can be installed using : conda install -c conda-forge ipopt or pip install ipopt
 
-This is followed by declaring a PYOMO Concrete Model, defining the variables, declaring the constraints using the parameters defined within “Input Variables” of the Node, and defining the objective function. (Lines (5) to (11)).
+Once the model is solved, the values of decision variables "x1", "x2", "y" are assigned to the Node Output Variables in lines (14) to (16).
 
-Lines (12) to (16) are used to access the solver, and solve the model via GAMS executable.
-
-Once the model is solved, the values of decision variables "x1", "x2", "y" are assigned to the Node Output Variables in lines (17) to (19).
-
-6. Run the Flowsheet and check the decision variable values at optimum in the Node Output Variables.
+6. Click the Run button, Run the Flowsheet and check the decision variable values at optimum in the Node Output Variables.
```

Comment:
Click the Run button to run the python script and check the Node Output Variables section.